### ISSUE TITLE
File -> Open Projects from File System... does not work in an IDE running in opp_env

### ISSUE THREAD

**Issue by avarga** [omnetpp/opp_env#11]
Steps to reproduce:
1. Run: opp_env shell omnetpp-6.0.1
2. download and unpack e.g. inet-4.5-src.tgz in its samples directory
3. launch the IDE, choose samples/ as workspace, cancel the dialog that offers auto-importing samples and installing inet
4. choose File -> Open Projects from File System... 
5. in the dialog, select inet-4.5 as project to be imported, click Finish
6. inet-4.5 is NOT imported (dialog closes but workspace stays empty)

The same function works (INET is imported) when done with a standalone (non-opp_env) omnetpp-6.0.1 installation.
Also seems to work when opening a standalone installation's workspace from omnetpp within opp_env
Problem seems to be with the workspace created by omnetpp installed and running in opp_env.

**Comment by rhornig** [comment]
Indeed. I can reproduce this also with omnetpp 6.0.2 (based on eclipse 4.29). There is no log written at all to `.metadata`. Needs further investigation.
